server/src/test/java/org/elasticsearch/health/node/HealthInfoTests.java

```diff
@@ -29,7 +30,14 @@ protected Writeable.Reader<HealthInfo> instanceReader() {
     protected HealthInfo createTestInstance() {
         var diskInfoByNode = randomMap(0, 10, () -> tuple(randomAlphaOfLength(10), randomDiskHealthInfo()));
         var repositoriesInfoByNode = randomMap(0, 10, () -> tuple(randomAlphaOfLength(10), randomRepoHealthInfo()));
-        return new HealthInfo(diskInfoByNode, randomBoolean() ? randomDslHealthInfo() : null, repositoriesInfoByNode);
+        return new HealthInfo(
+            diskInfoByNode,
+            randomBoolean() ? randomDslHealthInfo() : null,
+            repositoriesInfoByNode,
+            (randomFloat() < 0.1)
```

Comment:
We have `rarely()` for these kinds of things, but it's generally not something we want to do. This pattern makes it harder to reproduce and there's usually not a great reason to do it. Is there a reason you don't want `randomBoolean()`? I think that should be fine here as the mutation method is quick, so there's no impact on runtime or anything like that.

Reply:
I was afraid we'd get an overabundance of `INDETERMINATE` that way, leaving all the other cases under-tested.